src/clojure/nrepl/elisions.clj

```diff
@@ -0,0 +1,44 @@
+(ns nrepl.elisions
```

Comment:
I wonder if we shouldn't put this namespace and rich-printer under something like `nrepl.print` or whatever. Might be useful to structure the code like this if we ever come up with new printers.

Reply:
Some short summary of the purpose of the ns and add `:added 0.6` metadata would be a nice addition as well.

Reply:
Btw, why is this a separate ns to begin with? I wonder if it wouldn't better to have some `soft-store` ns and everything else to be in the `rich-print` ns.

Reply:
`elisions` contains the most specific parts: the printer function (for pr-values) and the elision store strategy.

`nrepl.print.elisions` and `nrepl.print.elisions.rich-printer`?

Reply:
Yeah, that sounds good to me.